assembly/pom.xml

```diff
@@ -14,7 +14,7 @@
   -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>splice_machine-assembly-${envClassifier}</artifactId>
+    <artifactId>splice_machine-assembly</artifactId>
```

Comment:
do we not have to add a classifier at the top level here?

Reply:
I see that the pattern here is completely the same, so i wont re-post this question elsewhere. I guess what i am missing is: how does an artifact get its classifier?

Reply:
Some artifacts do not have classifiers that differentiate them on nexus. If there was a need to have a differentiation for the artifact, I either added a configuration to add the envClassifier or if they didn't have the maven-jar-plugin, it was added. 

For, splice_machine assembly is a little odd since it's included into the shaded jar. We stopped deploying it to Nexus after 1825.